client/reader/stream/post.jsx

```diff
@@ -171,6 +171,13 @@ export default class ReaderPostCardAdapterFluxContainer extends React.Component
 		FeedPostStore.off( 'change', this.updateState );
 	}
 
+	shouldComponentUpdate( nextProps, nextState ) {
+		const shouldUpdate = this.props !== nextProps ||
```

Comment:
sorry, each render of the **parent** will generate a new props object. most of this is just the state changing, so yeah, not seeing props changes is expected.